packages/fxa-auth-server/test/local/routes/password.js

```diff
@@ -71,6 +76,152 @@ describe('/password', () => {
+
+      const mockRequest = mocks.mockRequest({
+        log: mockLog,
+        payload: {
+          email: TEST_EMAIL,
+          metricsContext: {
+            deviceId: 'wibble',
+            flowId:
+              'F1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF103',
+            flowBeginTime: Date.now() - 1,
+          },
+        },
+        query: {},
+        metricsContext: mockMetricsContext,
+      });
+      return runRoute(
+        passwordRoutes,
+        '/password/forgot/send_otp',
+        mockRequest
+      ).then((response) => {
+        assert.equal(
+          mockDB.accountRecord.callCount,
+          1,
+          'db.accountRecord was called once'
+        );
+        sinon.assert.calledOnce(mockRedis.set);
+
+        // an eight digit code was set
+        // TODO FXA-7852 check that the same code was pass to the email
+        assert.match(mockRedis.set.args[0][1], /\d{8}/);
+
+        assert.equal(
+          mockRequest.validateMetricsContext.callCount,
+          1,
+          'validateMetricsContext was called'
+        );
+
+        assert.equal(mockMetricsContext.setFlowCompleteSignal.callCount, 1);
+        const args = mockMetricsContext.setFlowCompleteSignal.args[0];
+        assert.lengthOf(args, 1);
+        assert.equal(args[0], 'account.reset');
+
+        assert.equal(
+          mockLog.flowEvent.callCount,
+          2,
+          'log.flowEvent was called twice'
+        );
+        assert.equal(
+          mockLog.flowEvent.args[0][0].event,
+          'password.forgot.send_otp.start',
+          'password.forgot.send_otp.start event was logged'
+        );
+        assert.equal(
+          mockLog.flowEvent.args[1][0].event,
+          'password.forgot.send_otp.completed',
+          'password.forgot.send_otp.completed event was logged'
+        );
+      });
+    });
+    it('returns an error when not enabled', () => {
+      const configDisabled = {
+        ...mockConfig,
+        passwordForgotOtp: { enabled: false },
+      };
+      const passwordRoutes = makeRoutes({
+        config: configDisabled,
+        customs: mockCustoms,
+        db: mockDB,
+        mailer: mockMailer,
+        metricsContext: mockMetricsContext,
+        log: mockLog,
+        authServerCacheRedis: mockRedis,
+        statsd: mockStatsd,
+      });
+
+      const mockRequest = mocks.mockRequest({
+        log: mockLog,
+        payload: {
+          email: TEST_EMAIL,
+          metricsContext: {
+            deviceId: 'wibble',
+            flowId:
+              'F1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF103',
+            flowBeginTime: Date.now() - 1,
+          },
+        },
+        query: {},
+        metricsContext: mockMetricsContext,
+      });
+      return runRoute(
+        passwordRoutes,
+        '/password/forgot/send_otp',
+        mockRequest
+      ).catch((response) => {
+        assert.equal(response.errno, error.featureNotEnabled().errno);
+      });
+    });
```

Comment:
Do we expect a certain error if the email doesn't exist?

Reply:
hmm I'll check the send_code endpoint to see its response for that.  I didn't see an error being explicitly returned/thrown in it for that state.

Reply:
ah `db.accountRecord` throws an `errors.unkownAccount`.